config/locales/en/lookup.yml

```
@@ -10,8 +10,10 @@ en:
         input_error: Enter a valid postcode
         error_description: Enter a valid postcode
     lookup:
-      input_label: Enter your postcode
-      hint_text: For example LS1 1UR
+      input_label: Enter a full postcode 
+      hint_text: |
+        <p class="govuk-hint"> We use this to give you more accurate information about the area.</p>
+        <p class="govuk-hint"> For example, LS1 1UR</p>
```

```suggestion
      hint_text: |
        We use this to give you more accurate information about the area.<br/>
        For example, LS1 1UR
```

I think this is probably best way of going about it